build_tools/travis/install.sh

```diff
@@ -38,6 +45,8 @@ make_conda() {
     if [ $TRAVIS_OS_NAME = "osx" ]
 	then
 		fname=Miniconda3-latest-MacOSX-x86_64.sh
+        # we need to install a version on clang which supports OpenMP
+        TO_INSTALL="$TO_INSTALL llvm-openmp clang"
```

Comment:
Maybe instead of using brew installed llvm-openmp and clang for the regular CI jobs we should stick to Apple clang (aliased as gcc / g++ by default under macOS) and enable openmp support by brew installing libomp and setting up the required environment variables as we do for the wheels.

I would rather have our regular CI build and test similar binaries as those we ship on PyPI.

Reply:
done